TestProjects/ShaderGraph/Assets/CommonAssets/Editor/ImportUpdateTests.cs

```diff
@@ -72,14 +72,15 @@ public void ImportMulti(string directory)
             string targetDir = Application.dataPath + "/Testing/ImportTests/" + dirName;
             try
             {
+                // pause asset database, until everything is copied
                 AssetDatabase.StartAssetEditing();
                 DirectoryCopy(sourceDir, targetDir, true, true);
             }
             finally
             {
                 AssetDatabase.StopAssetEditing();
             }
-
+//            AssetDatabase.Refresh();
```

Comment:
Does this need to be here commented out?

Reply:
I can strip it -- this was a case of fixing a bug in the test code, that was already fixed (though in a slightly different way) in master -- I commented out the colliding code here, but it can be removed (this test passes)